### DISCUSSION TITLE
Feature branches naming conventions

### DISCUSSION THREAD

**Discussion by Abhishek-Sathe-Tafe** [ZiyangWang001007081/Tafe230243#1] [Q&A] [answered]
Hello,

Just confirming if we are following the standard naming conventions for feature branches to maintain uniformity. Eg. feature_1.1_mortgage_calculator

**Comment by ZiyangWang001007081** [comment]
Hello Abhishek,

I think in our Git workflow practice (attached a screenshot below) we have defined the branches names as feature_Mortage and feature_Currency, but I'm happy to use feature_1.1_mortgage_calculator and feature_1.2_currency_calculator.
![230243GitWorkflow](https://github.com/ZiyangWang001007081/Tafe230243/assets/151332477/c793fb70-1455-47bf-b538-3fd91de2118d)

**Reply by Abhishek-Sathe-Tafe** [reply, marked as answer]
Nah if that's the case. Let's go with feature_Mortgage & feature_Currency
